setup/CMakeLists.txt

```diff
@@ -94,9 +94,6 @@ ExternalProject_Add(grpc
     -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
     -DCMAKE_POSITION_INDEPENDENT_CODE=on
     -DBUILD_SHARED_LIBS=on
-    -DgRPC_ABSL_PROVIDER=package
-    -DgRPC_PROTOBUF_PROVIDER=package
-    -DgRPC_SSL_PROVIDER=package
```

Comment:
1. If you omit `-DgRPC_SSL_PROVIDER=package`, gRPC will build and install Boring-SSL, which is apparently incompatible with the Tofino SDE.
2. Why not just update the versions of Abseil and Protobuf to be compatible with the version of gRPC? You're sacrificing our ability to patch Abseil or Protobuf independently in case of a security vulnerability.

Reply:
Ohh I was not aware of the first case. On lot of Validation systems, older version of abseil or protobuf were available  and having PROVIDER=package means use the available packages which will cause failure if version requirements are not satisfied. When you say update the version of Abseil and Protobuf, are we expecting the user to do this manually?

Reply:
It actually is not a Tofino incompatibility problem. We will run into issues compiling target-utils which houses the tdi_python cpython repo. Python explicitly states that boring-ssl is not supported and incompatible.

Reply:
https://peps.python.org/pep-0644/#boringssl

Reply:
Added it back